src/main/java/com/openshift/jenkins/plugins/pipeline/ParamVerify.java

```diff
@@ -37,62 +37,45 @@ public static void updateDSLBaseStep(Map<String, Object> arguments, OpenShiftBas
-            throws IOException, ServletException {
+    public static FormValidation doCheckBldCfg(@QueryParameter String value) {
         if (value.length() == 0)
             return FormValidation.error("You must set a BuildConfig name");
         return FormValidation.ok();
     }
     
-    public static FormValidation doCheckNamespace(@QueryParameter String value)
-            throws IOException, ServletException {
+    public static FormValidation doCheckNamespace(@QueryParameter String value) {
         if (value.length() == 0)
             return FormValidation.warning("Unless you specify a value here, the default namespace will be used; see this field's help or https://github.com/openshift/jenkins-plugin#common-aspects-across-the-rest-based-functions-build-steps-scm-post-build-actions for details");
         return FormValidation.ok();
     }
     
-    public static FormValidation doCheckToken(@QueryParameter String value)
-    		throws IOException, ServletException {
+    public static FormValidation doCheckToken(String value) {
     	if (value.length() == 0)
     		return FormValidation.warning("Unless you specify a value here, the default token will be used; see this field's help or https://github.com/openshift/jenkins-plugin#common-aspects-across-the-rest-based-functions-build-steps-scm-post-build-actions for details");
     	return FormValidation.ok();
     }
-        
-    
+
     public static FormValidation doCheckDestTagToken(@QueryParameter String value)
     		throws IOException, ServletException {
     	if (value.length() == 0)
     		return FormValidation.warning("A token is only needed if the new tag is targeted for a different project than the project containing the current tag");
     	return FormValidation.ok();
     }
     
-    public static FormValidation doCheckCheckForTriggeredDeployments(@QueryParameter String value)
-            throws IOException, ServletException {
-        if (value.length() == 0)
-            return FormValidation.error("Please set trigger check");
-        try {
-        	Boolean.parseBoolean(value);
-        } catch (Throwable t) {
-        	return FormValidation.error(t.getMessage());
-        }
-        return FormValidation.ok();
-    }
-
-    public static FormValidation doCheckCheckForWaitTime(@QueryParameter String value)
-            throws IOException, ServletException {
-        if (value.length() == 0)
+    public static FormValidation doCheckForWaitTime(String value) {
+        value = (value==null)?"":value.trim();
+        if ( value.isEmpty() ) // timeout will fallthrough to default
             return FormValidation.ok();
         try {
         	Long.parseLong(value);
         } catch (Throwable t) {
-        	return FormValidation.error(t.getMessage());
+            return FormValidation.ok( "Non-numeric value specified. During execution, an attempt will be made to resolve [%s] as a build parameter or Jenkins global variable.", value );
```

Comment:
I was originally thinking `warning` for this during our IRC, but I've swung around.  Let's keep with ok("...")